addons/website_sale_coupon/controllers/main.py

```diff
@@ -23,10 +23,10 @@ def payment(self, **post):
         return super(WebsiteSale, self).payment(**post)
 
     @http.route(['/shop/cart'], type='http', auth="public", website=True)
-    def cart(self, **post):
+    def cart(self, access_token=None, revive='', **post):
         order = request.website.sale_get_order()
         order.recompute_coupon_lines()
-        return super(WebsiteSale, self).cart(**post)
+        return super(WebsiteSale, self).cart(access_token, revive, **post)
```

Comment:
```suggestion
        return super(WebsiteSale, self).cart(access_token=access_token, revive=revive, **post)
```

Reply:
I have added the CLA md file. But the check on CLA is still failing. What else do I need to do?

Reply:
when you commit with Github, it doesn't use your email and so your CLA doesn't match.
Eg: 45359880+axelpriem@users.noreply.github.com

And, could you ammend your commit message to follow guidelines : https://www.odoo.com/documentation/14.0/reference/guidelines.html#commit-message-structure

thank you

Reply:
I have squashed commits and modified the message.
But I don't understand what I have to change for the CLA matching of my e-mail address. Could you explain please?

Reply:
you need to commit with the same email than in your cla...

`git config --list --show-origin | grep user.email`

This email doesn't match any CLA
![image](https://user-images.githubusercontent.com/4848797/105704412-0c461600-5f0f-11eb-966f-acea09d8b02b.png)

Btw it is probably a wrong idea to remove others users from cla ;) 
If they contributed in the past, they probably need to stay in the cla.
Maybe @mart-e can explain the best practice to clean cla with developer if it was really your intention (some doubts)

Reply:
OK, my email address in Github was not configured correctly, I didn't know that. It's fixed now.
I have changed the CLA file so the already present developers are not removed. This was indeed NOT my intention! I checked for an existing file before, but couldn't find it. Guess I overlooked it :-)
Thank you very much for signalling this to me!
All checks have now passed :-)

Reply:
You still need to amend your first commit with --author= to update it.
And while you are doing it, can you please amend your commit message for cla following guidelines ^^

> [CLA] for signing the Odoo Individual Contributor License;


thank you

Reply:
And if you can apply the suggested change to make named params clear ;) 

```py
        return super(WebsiteSale, self).cart(access_token=access_token, revive=revive, **post)
```

Reply:
Just need to squash forst and last commit now, and it hsould be ok :1st_place_medal: